packages/augur-ui/src/services/augursdk.ts

```diff
@@ -45,7 +41,10 @@ export class SDK {
       account
     );
 
-    const connector = (isMobileSafari() ? new SEOConnector(): new WebWorkerConnector());
+    const searchValues = history ? parseQuery(history.location.search) : {};
+    const { sdk_endpoint } = searchValues;
+
+    const connector = (isMobileSafari() ? new SEOConnector() : sdk_endpoint ? new WebsocketConnector(sdk_endpoint) : new WebWorkerConnector());
```

Comment:
This looks right.

I'd remove the ?: And use nornal ifs for this just for readability.